### DISCUSSION TITLE
[Help]: Can you put a Venmo username on the repo (for donations)?

### DISCUSSION THREAD

**Discussion by znwhite** [davidjerleke/embla-carousel#386] [Help]
I went to donate to you but PayPal is asking me to make an account when I submit payment, even when I select credit/debit card on KoFi. It would be a lot easier for some people if you also added a Venmo. Thanks!

**Comment by davidjerleke** [comment]
Hi @znwhite,

First of all, thank you very much for even considering supporting this project! It helps me improve the library and the documentation.

I tried getting Venmo but unfortunately it’s not available in Sweden (where I live). I don’t know if there are good alternatives out there that simplifies supporting this project.

Best,
David
